app/models/concerns/aliased_index.rb

```diff
@@ -9,7 +9,7 @@ module AliasedIndex
 
   module ClassMethods
     def timestamped_index_name
-      [base_name, Time.current.to_s(:number)].join('-')
+      [base_name, Time.current.to_s].join('-')
```

Comment:
Not sure if this will matter (but to be safe I'm going to assume it _does_), but this change will change the way this index name is named.  We'll go from something like:

```
"20240513174921"
```

to:
```
"2024-05-13 17:49:06 UTC"
```

The way to keep this identical without throwing an `ArgumentError` would be to change this to:

```
Time.current.to_formatted_s(:number)
```

Reply:
@lorawoodford  - I tried that as well but then breaks several other tests with the importers.

Reply:
@lorawoodford  - Yeah, that makes sense. I didn't really deep dive into dependent gems on this but would make sense an old version may not support. Will give that ago next.